public/app/workflow/workflow.admin.tpl.html

```diff
@@ -0,0 +1,55 @@
+<!--
+ Copyright 2016, RadiantBlue Technologies, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<!DOCTYPE html>
+<html lang="en" ng-app="SAKapp" ng-strict-di>
+<head>
+    <meta charset="UTF-8">
+    <title>Workflow Admin Info</title>
+    <link rel="stylesheet" href="../../css/bootstrap.css">
+
+
+</head>
+<body>
+
+<div ng-controller="WorkflowAdminController" class="container" ng-init="getStatus()">
+    <div class="row">
+        <h1 class="alert alert-info">Workflow Admin Info</h1>
+        <p>Display status info about Workflow Service</p>
+        <div>
+            <table>
+                <tr><td style="width:210px">Last Started:</td><td style="width:250px">{{adminData.starttime}}</td><td>({{getUptime(adminData.starttime)}})</td></tr>
```

Comment:
Talked to mpg and he said it will indeed be starttime.  This service shall behave as the others.  I will leave it in as this for now since he is going to fix the service side of things.